src/helpers.js

```diff
@@ -510,6 +435,8 @@ export default function Helpers(mpInstance) {
     this.decoded = utils.decoded;
     this.returnConvertedBoolean = utils.returnConvertedBoolean;
     this.parseStringOrNumber = utils.parseStringOrNumber;
+    this.generateHash = utils.generateHash;
+    this.generateUniqueId = utils.generateUniqueId;
```

Comment:
I'm fine with making a sustained engineering task to refactor fully remove these utility methods from Helpers. My goal is to make the scope as small as possible for refactors so that we don't impact any legacy functionality if at all possible.

Reply:
https://go.mparticle.com/work/SQDSDKS-5239